backend/gradle/dependency-locks/runtimeClasspath.lockfile

```diff
@@ -4,10 +4,10 @@
 antlr:antlr:2.7.7
 ch.qos.logback:logback-classic:1.2.3
 ch.qos.logback:logback-core:1.2.3
-com.azure:azure-core-http-netty:1.9.0
-com.azure:azure-core:1.14.1
-com.azure:azure-storage-common:12.11.0-beta.2
-com.azure:azure-storage-queue:12.9.0-beta.2
+com.azure:azure-core-http-netty:1.7.1
```

Comment:
This seems bad?

Reply:
Seems to have happened as a result of needing to use the `spring-cloud-dependencies` BOM to get the wiremock annotations to work properly, [see here](https://cloud.spring.io/spring-cloud-contract/1.1.x/single/spring-cloud-contract.html#_spring_cloud_contract_wiremock)
![image](https://user-images.githubusercontent.com/9121162/122287632-02b15080-cea6-11eb-81f3-11b2470e9c87.png)
@emmastephenson and I are trying to find a way around this, downgrading our azure deps is not desirable!

Reply:
Fixed this by updating `spring-cloud-dependencies` to `2020.0.3` and by forcing resolution of the runtime azure deps. Lockfiles are unchanged from `main` now

Reply:
Scratch that - it turns out `2020.0.3` breaks the test we are using WireMock in. Reverting to `Hoxton.SR11` along with some more forced resolutions results in a change set that only upgrades or adds dependencies, and the test passes.